src/i18n/en-US/index.ts

```diff
@@ -361,8 +361,10 @@ export default {
       faucetNextRequest: 'Time left until the next request',
       countDown: '{hrs} hrs {mins} mins {secs} secs',
       whatIsFaucet: ' What is faucet and how does it help you?',
+      faucetBalance: 'Faucet Balance: {amount} {symbol}',
       faucetIntro:
         'There is a small fee for each transaction, and this is paid using {symbol} tokens. If you have no {symbol} in your account, you cannot send any tokens. Faucet sends enough {symbol} to cover the transaction fee.',
+      faucetDriedOut: 'Faucet id dried out. Please report to one of our team members',
```

Comment:
Maybe to change message and put link to Discord here. It may be difficult for some users to find team members contact.

Reply:
I like that!

'Faucet id dried out. Please report to one of our team members in Discord' should be good

Permanent link: https://discord.gg/5mY3Rxu85P

Reply:
Updated!
GIF:
https://gyazo.com/d4bffa54965a450014f3ced1e116b4ec